libs/full/actions_base/include/hpx/actions_base/traits/action_stacksize.hpp

```diff
@@ -18,4 +19,8 @@ namespace hpx { namespace traits {
         static constexpr threads::thread_stacksize value =
             threads::thread_stacksize::default_;
     };
+
+    template <typename Action>
+    HPX_INLINE_CONSTEXPR_VARIABLE threads::thread_stacksize action_stacksize_v =
```

Comment:
You don't have to change it in this PR, but this just reminded me: we can use `inline constexpr` directly in these situations now. The macro always expands to that. A global find-and-replace would  be appropriate separately.

Reply:
See #5658